config/ext_jenkins-weekly.yaml

```
@@ -0,0 +1,146 @@
+serviceAccount:
+  create: true
+  name: jenkins-controller
+serviceAccountAgent:
+  create: false
+rbac:
+  create: true
+  readSecrets: true
+persistence:
+  enabled: true
+  size: 50Gi
+agent:
+  componentName: "agent"
+networkPolicy:
+  enabled: true
+  internalAgents:
+    allowed: true
+    namespaceLabels:
+      name: "jenkins-weekly"
+controller:
+  image: jenkinsciinfra/jenkins-weekly
+  tag: 0.50.0-2.341
+  pullPolicy: IfNotPresent
+  resources:
+    limits:
+      cpu: "2"
+      memory: "4Gi"
+    requests:
+      cpu: "2"
+      memory: "4Gi"
+  probes:
+    startupProbe:
+      initialDelaySeconds: 120
+    livenessProbe:
+      initialDelaySeconds: 120
+    readinessProbe:
+      initialDelaySeconds: 120
+  testEnabled: false
+  podSecurityContextOverride:
+    runAsNonRoot: true
+    runAsUser: 1000
+    supplementalGroups: [1000]
+  overwritePlugins: true
+  serviceType: "ClusterIP"
+  javaOpts: >
```

I prefer it with this:
```suggestion
  javaOpts: >-
```

and then arg per line?

see https://github.com/hmcts/cnp-flux-config/blob/master/apps/jenkins/jenkins/jenkins.yaml#L43-L66